tests/js/setup.ts

```diff
@@ -189,6 +189,20 @@ const routerFixtures = {
     ...params,
   }),
 
+  routeComponentProps: <RouteParams = {orgId: string; projectId: string}>(
```

Comment:
Is this different from what comes back from the `initializeOrg` thing?

Reply:
No, pretty much same thing, but there are tests that don't call initializeOrg and calling TestStubs.routeComponentProps() is more convenient than being forced to create org